README.md

```diff
@@ -1,100 +1,37 @@
-# River
+<p align="center">
+    <a href="https://towns.com">
+        <picture>
+            <source media="(prefers-color-scheme: dark)" srcset="assets/towns_dark.png">
+            <img width="240px" src="assets/towns.png" alt="Towns Protocol - a permissionless, decentralized end to end encrypted chat network" />
+        </picture>
+    </a>
+</p>
+<br/>
+<p align="center">
+  <a href="https://docs.towns.com"><img src="https://img.shields.io/badge/docs-read_the_docs-blue" alt="Read Towns Docs"></a>
+   <a href="https://docs.towns.com"><img src="https://img.shields.io/badge/governance-purple" alt="Towns Governance"></a>
+    <a href="https://docs.towns.com/grants/grants-program"><img src="https://img.shields.io/badge/grant-apply_for_funding-green" alt="Towns Grant"></a>
+    <a href="https://docs.towns.com/run/introduction"><img src="https://img.shields.io/badge/host_a_node-white" alt="Host a Towns Stream Node"></a>
```

Comment:
should we have this on the readme?

Reply:
I think just linking to the high level domains for now is enough.

Reply:
ok so I'll remove the `host a node` and `towns grant`